roxie/ccd/ccdserver.cpp

```diff
@@ -5210,11 +5219,15 @@ class CRemoteResultAdaptor : implements IEngineRowStream, implements IFinalRoxie
             else
             {
                 unsigned timeNow = msTick();
-                if (!anyActivity && !localAgent && (timeNow-lastActivity >= timeout))
+                if (acknowledgeAllRequests)
+                {
+                    retryPending(checkInterval);
+                }
+                else if (!anyActivity && !localAgent && (timeNow-lastActivity >= checkInterval))
                 {
                     lastActivity = timeNow;
-                    activity.queryLogCtx().CTXLOG("Input has stalled for %u ms - retry required?", timeout);
-                    retryPending();
+                    activity.queryLogCtx().CTXLOG("Input has stalled for %u ms - retry required?", checkInterval);
```

Comment:
could this get logged too often ?  (now that checkInterval is 1/10th of timeout)